.github/workflows/docker.yml

```diff
@@ -270,7 +270,14 @@ jobs:
         run: |
           scripts/install_gitea.sh
 
-      - name: pre e2e
+      - name: Generate build ID
+        id: uuid
```

Comment:
UUID has a very precise meaning and this doesn't seem to be that. https://en.wikipedia.org/wiki/Universally_unique_identifier